testbed/tests/window/test_window.py

```diff
@@ -142,13 +144,92 @@ async def test_move_and_resize(main_window, main_window_probe, capsys):
         finally:
             main_window.content = orig_content
 
-    async def test_full_screen(main_window, main_window_probe):
-        """Window can be made full screen"""
-        main_window.full_screen = True
-        await main_window_probe.wait_for_window("Full screen is a no-op")
+    @pytest.mark.skipif(
+        toga.platform.current_platform == "iOS", reason="Not implemented on iOS"
+    )
+    async def test_window_state_maximized(main_window, main_window_probe):
+        """Window can have maximized window state"""
+        assert main_window_probe.get_window_state() == WindowState.NORMAL
+        main_window.state = WindowState.MAXIMIZED
+        await main_window_probe.wait_for_window("WindowState.MAXIMIZED is a no-op")
+        assert main_window_probe.get_window_state() == WindowState.NORMAL
```

Comment:
Again - I know I've said on multiple past reviews that these tests need to test every transition *from* every state *to* every state - and, more than that, it needs to be clear that this is what is being tested. 

You have the "matrix" test further down the page; that test should be *first*, as it is the most important functional test. 

Once you've got that test, it's not clear what is being added by other tests (like this one). We're already validating that the window can be maximized, from every other state. Why do we need an explicit "can be maximized from NORMAL test?